src/lib.rs

```diff
@@ -186,16 +188,18 @@ pub enum PickMethod {
 
 /// Marks an entity to be used for picking
 pub struct PickSource {
-    groups: Option<Vec<Group>>,
-    pick_method: PickMethod,
-    cursor_events: EventReader<CursorMoved>,
+    pub groups: Option<Vec<Group>>,
+    pub pick_method: PickMethod,
+    pub always_on: bool,
```

Comment:
The `always_on` variable is only used for the `PickMethod::CameraCursor`, could you move it into that variant definition?

In addition To be a bit more self documenting, I would prefer an enum instead of a bool:
```rust
enum update_picks {
    always,
    on_mouse_event,
}
```

Reply:
Done, yes you are right.

I did not make it a default behavior though, will make it if you are certain we make it so. Thanks.

Reply:
The CI fails because the master branch is pointed to the bevy master, how should we deal with this?

Reply:
Looks like PR #69 will fix the CI issue. I'll merge that and we can re-run the tests.